packages/building-rollup/README.md

```diff
@@ -68,15 +68,17 @@ To illustrate this, here is an example of scripts which our rollup config will *
 {
   "scripts": {
     "build": "rimraf dist && rollup -c rollup.config.js",
-    "start:build": "http-server dist -o",
-    "watch:build": "rimraf dist && rollup --watch -c rollup.config.js & http-server dist -o",
+    "start:build": "http-server dist -o -c-1",
+    "watch:build": "rimraf dist && rollup --watch -c rollup.config.js & http-server dist -o -c-1",
   }
 }
 ```
 - `build` builds your app and outputs it in your `dist` directory
 - `start:build` runs your built app from `dist` directory
 - `watch:build` builds and runs your app, rebuilding when input files change
 
+Note that caching is disabled by default, if you want to enable caching, remove `-c-1` from the `start:build` and `watch:build` scripts.
```

Comment:
No need for this note I think. Starts to become too much information. People wont use this in production

Reply:
true - my bad I suggested it....

Reply:
Removed 👍